lib/auth/auth.go

```diff
@@ -869,10 +871,15 @@ func (a *Server) GenerateUserAppTestCert(req AppTestCertRequest) ([]byte, error)
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
-	checker, err := services.FetchRoles(user.GetRoles(), a.Access, user.GetTraits())
+	accessInfo, err := services.AccessInfoFromUser(user, a)
```

Comment:
Should this be 
```suggestion
	accessInfo, err := services.AccessInfoFromUser(user, a.Access)
```

like it is below in `GenerateDatabaseTestCert`?